privileges-grantor/README.md

```diff
@@ -11,8 +11,7 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | instance\_name | Privileges Grantor instance name to identify resources in multi-instance deployments. | string | `""` | no |
 | lambda\_timeout | The time after which the lambda execution stops. | string | `"200"` | no |
 | memory | The amount of memory (in MiB) to be used by Lambda | string | `"512"` | no |
-| metastore\_events\_filter | List of metastore event types to be added to SNS filter. Supported format: `[ "CREATE_TABLE","ALTER_TABLE" ]` | list | `[ "CREATE_TABLE", "ALTER_TABLE" ]` | no |
-| database\_filter | List of database names to be added to SNS filter. Supported format: `[ "DB_NAME_1", "DB_NAME_2" ]` | list | n/a | yes |
```

Comment:
If we've removed something again it's major vesion bump. Didn't this work? Or have we decided to remove the feature?

Reply:
this is for simplification:

According to the docs the Apiary metastore events listener can emit messages with 4 message attributes: https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-events/sns-metastore-events/apiary-metastore-listener#message-attributes

The lambda itself only cares about messages with eventType of "CREATE_TABLE" and "ALTER_TABLE": https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-events/sns-metastore-events/apiary-metastore-consumers

we can further filter what messages we pass to the lambda using an SNS subscription filter. If we decide to add an input variable in the tf module for every message attribute - that's 4, but let's not forget that you can make an SNS filter that can match a prefix (i.e. dbName prefix) and that's another 4 permutations and probably another 4 input variables for the message attributes but this time in prefixed form. There are also a reverse form of the SNS filter where you can pass through all messages apart from ones that "match" a certain pattern. All in all it is very difficult to capture this with terraform variables and generate a valid JSON in the end. The simplest option is to allow the user to pass this JSON directly to the module as a string via a new variable (and we have a default value as well)

Reply:
Yeah. I also faced same issue when I wrote it first time. Good to know that this was possible with >>JSON tag.